src/imf.jl

```diff
@@ -0,0 +1,78 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _imf{T<:AbstractFloat}(mass::AbstractVector{T}, expon::AbstractVector{T},
+                                mass_range::AbstractVector{T})
+    ne_comp = length(expon)
+    if length(mass_range) != ne_comp + 1
+        println("Length of array mass_range is not one more than that of expon")
+        return zeros(mass)
+    end
+    integ = ones(T, ne_comp)
```

Comment:
The initial value of `integ` doesn't look to be relevant.  You can save some nanoseconds by initializing the vector with `Vector{T}(ne_comp)`.